cmd/iam_apikey_create.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type apiKeyCreateItemOutput egoscale.APIKey
+
+func (o *apiKeyCreateItemOutput) toJSON()  { outputJSON(o) }
+func (o *apiKeyCreateItemOutput) toText()  { outputText(o) }
+func (o *apiKeyCreateItemOutput) toTable() { outputTable(o) }
+
+// apiKeyCreateCmd represents an API key creation command
+var apiKeyCreateCmd = &cobra.Command{
+	Use:     "create <description>",
+	Short:   "Create an API key",
+	Aliases: gCreateAlias,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return cmd.Usage()
+		}
+
+		ops, err := cmd.Flags().GetStringSlice("operations")
+		if err != nil {
+			return err
+		}
+
+		resp, err := cs.RequestWithContext(gContext, &egoscale.CreateAPIKey{
+			Description: args[0],
+			Operations:  strings.Join(ops, ","),
+		})
+		if err != nil {
+			return err
+		}
+
+		apiKey := resp.(*egoscale.APIKey)
+		o := apiKeyCreateItemOutput(*apiKey)
+		return output(&o, err)
+	},
+}
+
+func init() {
+	apiKeyCreateCmd.Flags().StringSliceP("operations", "o", []string{}, "API key operation")
```

Comment:
```suggestion
	apiKeyCreateCmd.Flags().StringSliceP("operations", "o", []string{}, "API key allowed operation")
```

Is this flag a multiple-values flag (e.g. `exo iam create -o op1 -o op2 -o opN`) or a single value with multiple values comma-separated (e.g. `exo iam create -o op1,op2,opN`)? In the first case the long option should read `--operation` (because 1 operation per flag), otherwise `--operations`.

Reply:
You can do both.

Reply:
Well in this case I'd opt for `--operation` instead of `--operations`.